app/controllers/EditionsController.scala

```diff
@@ -110,17 +130,28 @@ class EditionsController(db: EditionsDB,
   }
 
   def listIssues(edition: Edition) = EditEditionsAuthAction { req =>
-    Try {
+    val params = Try {
       val dateFrom = req.queryString.get("dateFrom").map(_.head).get
       val dateTo = req.queryString.get("dateTo").map(_.head).get
       (LocalDate.parse(dateFrom), LocalDate.parse(dateTo))
```

Comment:
Now that I think about it, this might be a another good usecase for [Validated](https://typelevel.org/cats/datatypes/validated.html) – I guess it’s only two dates, but if there’s a problem with both of them it’d be nice to be able to report the two errors instead of just the first.

Reply:
Agreed! I do wonder whether getting these sorts of libraries in early means there's not this general slightly suboptimal approach to validation across the board. I'm 🚢  now due to time constraints (thereby extending this problem!) 😅

Reply:
Yup, no problem! Definitely more of a future option to look at than something to do in this PR.